shared/src/commonMain/kotlin/co/nimblehq/blisskmmic/di/koin/modules/UseCaseModule.kt

```diff
@@ -4,9 +4,12 @@ import co.nimblehq.blisskmmic.domain.usecase.LogInUseCase
 import co.nimblehq.blisskmmic.domain.usecase.LogInUseCaseImpl
 import co.nimblehq.blisskmmic.domain.usecase.ResetPasswordUseCase
 import co.nimblehq.blisskmmic.domain.usecase.ResetPasswordUseCaseImpl
+import co.nimblehq.blisskmmic.domain.usecase.SurveyListUseCase
+import co.nimblehq.blisskmmic.domain.usecase.SurveyListUseCaseImpl
 import org.koin.dsl.module
 
 val useCaseModule = module {
     single<LogInUseCase> { LogInUseCaseImpl(get()) }
     single<ResetPasswordUseCase> { ResetPasswordUseCaseImpl(get()) }
+    factory<SurveyListUseCase> { SurveyListUseCaseImpl(get()) }
```

Comment:
I am curious that can't we use `single` here? 🤔

Reply:
I used `single` because I was worried about logout case where the old data might persist. With current implementation, it should be possible to use `single` now.